### DISCUSSION TITLE
[OP Stack Deployment] Deploying contracts to L1 failing

### DISCUSSION THREAD

**Discussion by Suhel-Kap** [ethereum-optimism/developers#294] [Deploying the OP Stack] [answered]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

 I'm following the create L2 rollup tutorial listed here https://docs.optimism.io/builders/chain-operators/tutorials/create-l2-rollup

When I run the "Deploy the L1 contracts" step's forge script command, it fails.

```
forge script scripts/Deploy.s.sol:Deploy --private-key $GS_ADMIN_PRIVATE_KEY --broadcast --rpc-url $L1_RPC_URL --slow
```

Then next when I ran the command I got this error
== Logs ==
Connected to network with chainid 11155420
Storing temp deployment data in /Volumes/T7/clamp-R&D/op-rollup-testnet/optimism/packages/contracts-bedrock/deployments/getting-started/.deploy
DeployConfig: reading file /Volumes/T7/clamp-R&D/op-rollup-testnet/optimism/packages/contracts-bedrock/deploy-config/getting-started.json
Deploying from Deploy
Deployment context: getting-started
Deploying a fresh OP Stack including SuperchainConfig
Deploying Safe
New SystemOwnerSafe deployed at 0xFc92CfA2DefBf4585fb176FA1e10507501Af5470
Setting up Superchain
Deploying AddressManager
AddressManager deployed at 0xd1f9DAEf5bbb3aC0Df10aB584850b8699c73910C
Deploying ProxyAdmin
ProxyAdmin deployed at 0x3F430dC7eefc6F0c76C9de8e9044599D253014f7
ProxyAdmin ownership transferred to Safe at: 0xFc92CfA2DefBf4585fb176FA1e10507501Af5470
Deploying ERC1967 proxy forSuperchainConfigProxy
at 0x1d03cd5a572a1E0dC634BFD85971130419D7Df9E
SuperchainConfig deployed at 0x3cbd67985B65B7A3fCEc7fDfDFF44aE5f4381b4E
Running chain assertions on the SuperchainConfig
Deploying ERC1967 proxy forProtocolVersionsProxy
at 0xFe2c41f523CEfda549193aa10c24fA59c8726682
Deploying ProtocolVersions implementation
ProtocolVersions deployed at 0x06575C2569466A0cb76f6d80812A2674Ef1bFFB8
Running chain assertions on the ProtocolVersions
Error:
script failed: failed parsing JSON: EOF while parsing a value at line 1 column 0

how should i resolve this, I simply followed the tutorial upto this point yet it failed

### Feedback

So the tutorial doesn't mention but I had to do `forge install` first, maybe you should add that in the tutorial

**Comment by k0de26** [comment, marked as answer]
It seems you missed a step in [Configure your network.](https://docs.optimism.io/builders/chain-operators/tutorials/create-l2-rollup#configure-your-network) 

After running the command `./scripts/getting-started/config.sh` 

Head over to your optimism/packages/contracts-bedrock/deploy-config 
directory in your Optimism mono-repo and make sure that `getting-started.json` file was created.
